Libraries/LibWeb/SVG/SVGClipPathElement.h

```diff
@@ -8,13 +8,14 @@
 
 #include <LibWeb/SVG/AttributeParser.h>
 #include <LibWeb/SVG/SVGElement.h>
+#include <LibWeb/SVG/SVGGraphicsElement.h>
 #include <LibWeb/SVG/SVGViewport.h>
 
 namespace Web::SVG {
 
-class SVGClipPathElement final : public SVGElement
+class SVGClipPathElement final : public SVGGraphicsElement
```

Comment:
could you make a separate commit that makes SVGClipPathElement inherit from SVGGraphicsElement